core/common/src/ElementProps.ts

```diff
@@ -668,4 +668,48 @@ export interface VersionedJSON<T> {
   version: ECVersionString;
   /** The JSON data. */
   data: T;
-}
\ No newline at end of file
+}
+
+/** Project-level properties hosted by a [ProjectInformationRecord]($backend) element.
+ * @beta
+ */
+export interface ProjectInformation {
+  /** A name that uniquely identifies this project within an organization. */
+  projectNumber?: string;
+  /** A human-readable display label for the project. Unlike [[projectNumber]], this needn't be unique. */
+  projectName?: string;
+  /** A human-readable description of the geographic location of the project - often, the name of a city or a street address. */
+  location?: string;
+}
+
+/** JSON representation of a [ProjectInformationRecord]($backend).
+ * @beta
+ */
+export type ProjectInformationRecordProps = ElementProps & ProjectInformation;
+
+/** Captures basic metadata about a [Sheet]($backend). This information is typically included in the Sheet's graphical representation, often
+ * as part of its title block.
+ * This information is stored in a [SheetInformationAspect]($backend).
+ * @beta
+ */
+export interface SheetInformation {
+  designedBy?: string;
```

Comment:
@Josh-Schifter @diegoalexdiaz I did not document these because they are not documented in BisCore schema. Please contribute documentation.

Reply:
For all four, I would write:
"A user supplied string/date that describes a sheet.  Suitable for use in the sheet's title block."